src/runtime/gecko/util.js

```diff
@@ -0,0 +1,41 @@
+export { documentReady, getResourceLinks, getMeta } from '../web/util';
+import { getLangRevisionMap, getLangRevisionTuple } from '../web/util';
+
+export function getXULResourceLinks(doc) {
+  return Array.prototype.map.call(
+    doc.querySelectorAll('messagebundle'),
+    el => el.getAttribute('src'));
+}
+
+export function getXULMeta(doc) {
```

Comment:
for now yes, we don't have any other place to get info about languages available for the given files yet.